netbox/utilities/views.py

```diff
@@ -88,15 +88,28 @@ def queryset_to_csv(self):
         Export the queryset of objects as comma-separated value (CSV), using the model's to_csv() method.
         """
         csv_data = []
+        custom_fields = []
 
         # Start with the column headers
         headers = ','.join(self.queryset.model.csv_headers)
+
+        # Add custom field headers
+        content_type = ContentType.objects.get_for_model(self.queryset.model)
+
+        for custom_field in CustomField.objects.filter(obj_type=content_type):
+            headers += ',cf_{}'.format(custom_field.name)
```

Comment:
It's a bit tidier to keep `headers` as a list and call `.append()` to add the custom field headers. You can move the `join()` logic below this to generate the string.